Autopilot/TelemetryManager/Telemetry_Manager.c

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file Telemetry_Manager.c
+ * @author Jingting Liu
+ * @date January 14th, 2021
+ */
+
+#include <stdio.h>
```

Comment:
Since we are using C++, <stdio.h> won't work (use <iostream> instead, though I don't think it's needed since we don't use any terminal output/input streams)

Reply:
@Pop0097 Hmm okay I'll remove this include, but I can still write this in C right?

Reply:
Anthony pointed out that he would like for the State machines to have a consistent structure. Since our other state machines use class objects, we'll need to use C++.